astrology.lic

```diff
@@ -234,7 +234,12 @@ class Astrology
         bput('open my telescope', 'extend your telescope')
         return observe(body)
       end
-      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'Roundtime')
+      result = bput('peer telescope', 'You peer aimlessly through your telescope', 'You see nothing regarding the future', 'The pain is too much', 'Roundtime')
+        if result =~ /The pain is too much/
+          bput('stow telescope', 'you put', "That can't be picked up")
+          wait_for_script_to_complete('safe-room')
+          return observe(body)
```

Comment:
This is how `pick.lic` handles this situation: https://github.com/rpherbig/dr-scripts/blob/master/pick.lic#L382-L384